### ISSUE TITLE
The optical flow ground truth is not available to download

### ISSUE THREAD

**Issue by Yaozhuwa** [uzh-rpg/DSEC#31]
In DSEC Dataset download page [https://dsec.ifi.uzh.ch/dsec-datasets/download/](https://dsec.ifi.uzh.ch/dsec-datasets/download/), The test set only has test_events.zip. The test_optical_flow.zip is not available. So I can't evaluate my training.
![image](https://user-images.githubusercontent.com/28787183/146636197-7433f8f0-2723-4cb8-a56f-932ccc9afa17.png)

**Comment by magehrig** [comment]
Hi @Yaozhuwa 

You can do a random split (I suggest a 1 to 5 ratio for the validation set) of the training set to train and validation set for experimenting with different approaches. Once you are satisfied, you should submit to the optical flow evaluation server: https://dsec.ifi.uzh.ch/uzh/dsec-flow-optical-flow-benchmark/. For the final submission, it is best to train on the full validation set.  Best to follow the advice I gave in another issue: https://github.com/uzh-rpg/E-RAFT/issues/1

We also evaluate our results with the evaluation server as you can see in the entry of the leaderboard. You can also submit and not make it public. Check out this issue with a similar question and some additional info: https://github.com/uzh-rpg/E-RAFT/issues/1
